clients/test/lightrider-entry-test.js

```diff
@@ -59,7 +59,7 @@ describe('lightrider-entry', () => {
     });
 
     it('specifies the channel as lr', async () => {
-      const runStub = jest.spyOn(Runner, 'run');
+      const runStub = jest.spyOn(Runner, 'gather');
```

Comment:
not that these spy tests are the best, but seems like these should be on the higher-level `lighthouse()` function now

Reply:
One of the tests verifies that the `config.settings.channel` is set to `lr` before `config` reaches `Runner`. I think there is still value in testing `Runner.gather` rather than `lighthouse()`.